applications/ParticleMechanicsApplication/python_scripts/particle_mechanics_analysis.py

```diff
@@ -66,14 +35,15 @@ def RunSolutionLoop(self):
         ## Analysis timer start
```

Comment:
Do you really need this. I thought you told me you deleted it. Anyways, not major.

Reply:
I thought to delete this, but then I felt that this is important to really control the performance, especially for my tests. So I brought it back.